src/Components/Results/Results.tsx

```diff
@@ -578,6 +557,15 @@ const Results = () => {
       { field: 'eligible', headerName: 'Eligible', flex: 1, type: 'boolean' },
       { field: 'has_benefit', headerName: 'Has Benefit', flex: 1, type: 'boolean' },
       { field: 'category', headerName: 'Category', flex: 1 },
+      {
+        field: 'toggle',
+        headerName: '',
+        flex: 0.15,
+        align: 'right' as GridAlignment,
+        renderCell: (params: GridRenderCellParams) => (
+          <CustomDetailPanelToggle {...params} id={params.id} field={params.field} rowNode={params.rowNode} />
```

Comment:
```suggestion
          <CustomDetailPanelToggle id={params.id} field={params.field} rowNode={params.rowNode} />
```

Reply:
Or you can remove `id={params.id} field={params.field} rowNode={params.rowNode}`